### PR TITLE
Address Security Vulnerabilities - Jan 2020

### PR DESCRIPTION
**Pull request by shirish-pampoorickal**
## Build Status
[Build Status](https://travis-ci.org/mavenlink/brainstem)
---

## Details

- [x] Bump [actionview](https://github.com/rails/rails/tree/master/actionview) from 5.2.4.1 to 5.2.4.2 / 6.0.2.1 => 6.0.2.2. Addresses [GHSA-65cv-r6x7-79hv](https://github.com/advisories/GHSA-65cv-r6x7-79hv)

- [x] Bump [nokogiri]() from 1.10.8 to 1.10.9. Addresses [GHSA-7553-jr98-vx47](https://github.com/advisories/GHSA-7553-jr98-vx47)

- [x] Bumps [rack](https://github.com/rack/rack) from 2.0.7 to 2.2.2. Addresses [GHSA-hrqr-hxpp-chr3](https://github.com/advisories/GHSA-hrqr-hxpp-chr3)

- [X] Bumps [loofah](https://github.com/flavorjones/loofah) from 2.3.0 to 2.4.0. Addresses [GHSA-c3gv-9cxf-6f57](https://github.com/advisories/GHSA-c3gv-9cxf-6f57)

### CODE REVIEW

**Comment by cpalafox** [comment]
There's something going on with the integration of Github and TravisCi, but the latest build triggered by sha `5c3cf67` can be found here https://travis-ci.org/github/mavenlink/brainstem/builds/666545830

File: .travis.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ services:
   - mysql
 
 before_install:
-  - gem install bundler
+  - gem install bundler -v "$(grep -A 1 "BUNDLED WITH" Gemfile.lock | tail -n 1)"
```
Comment (naganowl) [inline .travis.yml]: I'm wondering if there's a different CI/Ubuntu image we need to use

**Comment by a-bates** [comment]
There's a fair amount of changes between rack 2.0.7 and 2.2.2. Do any of those changes apply to us? https://github.com/rack/rack/blob/master/CHANGELOG.md#220---2020-02-08

**Comment by shirish-pampoorickal** [comment]
@a-bates I am relying on specs to verify that rack upgrade is good. I am also going to verify this on our own app to check if there's any issues

**Comment by a-bates** [comment]
ship it

**Comment by naganowl** [comment]
@shirish-pampoorickal is there a bigmaven branch that uses this?

**Comment by naganowl** [comment]
Awesome, let's merge this branch into master so we can update https://github.com/mavenlink/mavenlink/pull/19445

**Pull request merged**